schemas/server-common.json

```diff
@@ -24,6 +24,14 @@
       "minLength": 3,
       "maxLength": 44
     },
+    "datasetMember": {
+      "$anchor": "zoweDatasetMember",
+      "type": "string",
+      "description": "A 44-char all caps dotted ZOS name",
```

Comment:
Hi, I think this description is for dataset name, not for member name, it should be "A 1-8char all caps dataset member name"

Reply:
Yes! Curses